internal/cmd/gocdk/internal/static/_assets/resource/docstore/awsdynamodb.tf

```diff
@@ -0,0 +1,16 @@
+# TODO(rvangent): Add comments explaining.
+
+locals {
+  awsdynamodb_url = "dynamodb://${aws_dynamodb_table.table.name}?region=${local.aws_region}&partition_key=Key&sort_key=Key"
```

Comment:
I don't think you can use the same field for the partition and sort key. I think you can omit the sort key.

Reply:
OK, done.